cmd/openfish/handlers/auth.go

```diff
@@ -35,22 +35,77 @@ LICENSE
 package handlers
 
 import (
-	"github.com/ausocean/openfish/cmd/openfish/entities"
+	"fmt"
+
+	"github.com/ausocean/openfish/cmd/openfish/api"
+	"github.com/ausocean/openfish/cmd/openfish/services"
+	"github.com/ausocean/openfish/cmd/openfish/types/role"
 	"github.com/gofiber/fiber/v2"
```

Comment:
Order of imports should be standard library, then 3rd party, then ausocean packages.

Reply:
When I run `go fmt ./cmd/openfish` it reorders them into this order, so I would prefer not to change that.